Filecontent_indexhtml and css/index.html

```diff
@@ -0,0 +1,275 @@
+		<div class="work_contains">
+	   <img class="add_snaphoot1" src="./Images/Snapshoot2.png">
+
+	   <div class="info_work_cont">
+		 <h1 class="Add align-item">Multi-Post<h1><h1>stories</h1>
+	   </div>
+
+	   <div class="description_a_items ">
+		 <p class="company"> CANOPY </p>
+		 <div class="separator"></div>
+
+		 <p class="responsability"> Back End Dev </p>
+
+		 <div class="separator"></div>
+		 <p class="year"> 2015 </p>
+	   </div>
+
+	   <div class="projects_descriptions">
+		 <p> A daily selection of privately personalized reads; no accounts or sign-ups required. </p>
+	   </div>
+
+	  
+	   <div class="programming_icon">
+
+		<a href="#" target="_blank" id="html1"><img src="Images/html.png"> </a>
+
+		<a href="#" target="_blank" id="html1"><img src="Images/css.png"> </a>
+		
+		<a href="#" target="_blank" id="html1"><img src="Images/javascript.png"> </a>  
+	  </div>
+
+	</div>
+	   <button class="vizualization_projects"> See Project </button>
+	 </div>
+	 </section>
+
+	  <section class="contains_work_section">
+		<div class="work_contains">
+	   <img class="add_snaphoot1" src="./Images/Snapshoot3.png">
+
+	   <div class="info_work_cont">
+		 <h1 class="Add align-item">Tonic</h1>
+	   </div>
+
+	   <div class="description_a_items ">
+		 <p class="company"> CANOPY </p>
+		 <div class="separator"></div>
+
+		 <p class="responsability"> Back End Dev </p>
+
+		 <div class="separator"></div>
+		 <p class="year"> 2015 </p>
+	   </div>
+
+	   <div class="projects_descriptions">
+		 <p> A daily selection of privately personalized reads; no accounts or sign-ups required. </p>
+	   </div>
+        
+	   <div class="programming_icon">
+
+		<a href="#" target="_blank" id="html1"><img src="Images/html.png"> </a>
+
+		<a href="#" target="_blank" id="html1"><img src="Images/css.png"> </a>
+		
+		<a href="#" target="_blank" id="html1"><img src="Images/javascript.png"> </a>  
+	  </div>
+	  
+	   </div>
+	   <button class="vizualization_projects"> See Project </button>
+	 </div>
+	 </section>
+
+	 
+	 <section class="contains_work_section">
+		<div class="work_contains">
+	   <img class="add_snaphoot1" src="./Images/Snapshoot4.png">
+
+	   <div class="info_work_cont">
+		 <h1 class="Add align-item">Multi-Post</h1><h1>stories</h1>
+	   </div>
+
+	   <div class="description_a_items ">
+		 <p class="company"> CANOPY </p>
+		 <div class="separator"></div>
+
+		 <p class="responsability"> Back End Dev </p>
+
+		 <div class="separator"></div>
+		 <p class="year"> 2015 </p>
+	   </div>
+
+	   <div class="projects_descriptions">
+		 <p> A daily selection of privately personalized reads; no accounts or sign-ups required. </p>
+	   </div>
+
+	   <div class="programming_icon">
+
+		<a href="#" target="_blank" id="html1"><img src="Images/html.png"> </a>
+
+		<a href="#" target="_blank" id="html1"><img src="Images/css.png"> </a>
+		
+		<a href="#" target="_blank" id="html1"><img src="Images/javascript.png"> </a>  
+	  </div>
```

Comment:
Kindly check the margins of top, bottom, left and right are the same as the figma design. There is a blank empty space to the right side of the page as shown below. Kindly check and fix it to match the figma design. :pray: 

Your design 

![Screenshot from 2022-12-10 23-37-57](https://user-images.githubusercontent.com/65233860/206879629-97a495a1-f3e3-45ae-9537-6b69d8a2ddd4.png)


The figma design 

![Screenshot from 2022-12-10 23-38-10](https://user-images.githubusercontent.com/65233860/206879632-d15a3c1a-658e-4192-9dbe-740778d82c63.png)

Reply:
As the previous reviewer suggested, there still seems to be a blank empty space to the right side of the page as shown below. Kindly check and fix it to match the figma design. Kindly fix this.


![image](https://user-images.githubusercontent.com/20554583/206888939-ce27f5fc-6a41-41ef-a8d1-0b74e6288314.png)